lgtm.yaml

```diff
@@ -0,0 +1,29 @@
+path_classifiers:
+  test:
+    - libvast_test
+extraction:
+  cpp:
+    prepare:
+      packages:
+        - libssl-dev
+        - python-pip
+    after_prepare:
+      - mkdir -p $LGTM_WORKSPACE/latest-gcc-symlinks
+      - ln -s /usr/bin/g++-8 $LGTM_WORKSPACE/latest-gcc-symlinks/g++
+      - ln -s /usr/bin/gcc-8 $LGTM_WORKSPACE/latest-gcc-symlinks/gcc
+      - export BUILD_TYPE=Release
+      - export GIT=true
+      - export PATH=$LGTM_WORKSPACE/latest-gcc-symlinks:$PATH
+      - export PREFIX=/opt/tenzir
+    configure:
+      command:
+        - pip3 install cmake
+        - /opt/work/.local/lib/python3.6/site-packages/cmake/data/bin/cmake --version
```

Comment:
Those explicit paths seem to be brittle when it comes to version changes. Is there perhaps a cleaner approach based on virtual envs? (This assumes that the environment can be mutated between individual steps.)

Reply:
@mavam We could resolve the site package location dynamically, such as `$(python -m site --user-site)/cmake/data/bin/cmake --version`

Reply:
Seems better, albeit still noisy.